packages/shorebird_cli/test/src/executables/gradlew_test.dart

```diff
@@ -329,5 +329,110 @@ Make sure you have run "flutter build apk" at least once.''',
         },
       );
     });
+
+    group('exists', () {
+      late Directory tempDir;
+      setUp(() {
+        tempDir = setUpAppTempDir();
+      });
+
+      group('when gradlew does not exist', () {
+        test('returns false', () {
+          expect(gradlew.exists(tempDir.path), isFalse);
+        });
+      });
+
+      group('when gradlew exists', () {
+        group(
+          'when on unix based OSs',
+          () {
+            setUp(() {
+              File(
+                p.join(tempDir.path, 'android', 'gradlew'),
+              ).createSync(recursive: true);
+            });
+
+            test(
+              'returns true',
+              () {
+                expect(gradlew.exists(tempDir.path), isTrue);
+              },
+              testOn: 'linux || mac-os',
```

Comment:
Neat, I didn't know `OS1 || OS2` was supported syntax

Reply:
yeah, me neither, I just learned. Apparently `testOn` uses this package: https://pub.dev/packages/boolean_selector